jib-core/src/main/java/com/google/cloud/tools/jib/event/EventEmitter.java

```diff
@@ -16,30 +16,8 @@
 
 package com.google.cloud.tools.jib.event;
 
-import com.google.common.collect.ImmutableMultimap;
+/** Emits {@link JibEvent}s. */
+public interface EventEmitter {
```

Comment:
Not sure I understand why this is necessary, what might non-default implementations of this look like?

Reply:
For example, a no-op EventEmitter such as on BuildConfiguration line 55.